### ISSUE TITLE
TypeScript Support

### ISSUE THREAD

**Issue by frodo821** [oauth-io/oauth-js#105]
I recommend adding `index.d.ts` to this project. TypeScript has more users than CoffeeScript, so supporting typescript is a better option.

**Comment by jbatte47** [comment]
I agree, although apparently it's me and one other person that cares about this. I have to go find another OAuth proxy provider because working with TypeScript in my project is more important to me than sticking with OAuth.io.
